app/templates/email/notification_change.html

```diff
@@ -0,0 +1,6 @@
+<p>Dear {{ user.username }},</p>
+<p>This is an email notification of a change to your account.</p>
+<p>If you made this change, please ignore this email.</p>
+<p>If you believe someone has unauthorized access to your account, please contact us immediately.</p>
+<p>Sincerely,</p>
+<p>The Droids Support Services</p>
```

Comment:
Make sure that you add some space in between to improve readability

Reply:
The email is properly formatted, it just looks weird in html format